Essentials/src/messages.properties

```diff
@@ -165,7 +159,7 @@ haveBeenReleased=\u00a76You have been released.
 heal=\u00a76You have been healed.
 healDead=\u00a74You cannot heal someone who is dead\!
 healOther=\u00a76Healed\u00a7c {0}\u00a76.
-helpConsole=To view help from the console, type ?.
+helpConsole=To view help from the console, type ?
```

Comment:
This change doesn't seem consistent with the other messages. Perhaps `'?'.` would be more appropriate?

Reply:
I just deleted this to prevent users from entering ?. because then the command will not execute.

Reply:
I would change this to:
```properties
helpConsole=To view help from the console, type ''?''.
```